src/pages/RadicalCollaboration/Share/MediaInfo.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import useStyles from './styles.js';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Link from '../../../components/common/Link';
+
+const CtiLogoSection = () => {
+    const classes = useStyles();
+  return (
+    <Container className={classes.logoContainer}>
+        <img className={classes.logoBW} src='/images/OneColor_CTI_Logo.svg' alt='CTI logo' />
+        <img
+        className={classes.logoColor}
+          src='/images/FullColor_CTI_Logo.svg'
+          alt='CTI logo color'
+        />
+    </Container>
+  );
+};
+
+const ButtonSection = () => {
+  return (
+    <Grid
+      container
+      direction='column'
+      alignItems='center'
+      style={{ marginTop: '40px' }}
+    >
+      <Button
+        href='https://drive.google.com/drive/folders/1EORoPWmALLXJCfHyDXJemYpO5HvnMzPN'
+        target='blank'
```

Comment:
Should target have an underscore to point to a new window?
```suggestion
        target='_blank'
```